.gitignore

```diff
@@ -128,4 +128,10 @@ node_modules/
 
 # GraphViz
 *.png
-*.dot
\ No newline at end of file
+*.dot
+
+# src tarballs
+.src.tar.gz
+
+# OSX
+.DS_Store
```

Comment:
OSX files attempted to creep in during the refactoring.

Reply:
not me, i swear